frontend/src/components/AccountList/MainSectionAccountList.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import AccountListContainer from '../../container/AccountListContainer';
+
+export default class MainSectionAccountList extends Component {
+    // constructor() {
+    //     super();
+    //     this.state = {
+    //         accounts: []
+    //     };
+    // }
+
+    render() {
+        return <div>
+            <AccountListContainer/>
```

Comment:
Presentational components cannot wrap Container components. The container components should be root ones.